amlb/benchmark.py

```diff
@@ -404,6 +404,9 @@ def __init__(self, name, openml_task_id, test_server, fold, metrics, seed,
         self.git_info = git_info
         self.measure_inference_time = measure_inference_time
         self.ext = ns()  # used if frameworks require extra config points
+        if quantile_levels is None:
+            quantile_levels = [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]
```

Comment:
Is it possible to store the default `quantile_levels` in `config.yaml` instead of here?

Reply:
From my understanding it seems it would be fine under `benchmarks.defaults`, or do you think a different spot is more appropriate?

Reply:
Moved the default values to `benchmark.defaults` section of `config.yaml`.